e2e/vue3-vite-e2e/tests/component.spec.ts

```diff
@@ -9,10 +9,13 @@ import {
 jest.setTimeout(60000);
 
 describe('component e2e', () => {
+  beforeEach(() => {
+    ensureNxProject('nx-vue3-vite', 'dist/packages/vue3-vite');
```

Comment:
This command creates the temp directory, I moved it to `beforeEach` in all the E2E tests to make sure the repo doesn't get corrupted when E2E tests are run 😅